packages/dtx-web/src/lib/services/workspaceService.ts

```
@@ -0,0 +1,401 @@
+			uniqueName = `${baseName} (${counter})`;
+		}
+
+		return uniqueName;
+	}
+
+	/**
+	 * Get a large file from session memory
+	 */
+	static getLargeFile(workspaceName: string, fileName: string): File | undefined {
+		const fileKey = `${workspaceName}/${fileName}`;
+		return this.sessionLargeFiles.get(fileKey);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)